packages/sit-onyx/src/components/TestInput/TestInput.vue

```diff
@@ -34,12 +62,25 @@ const handleChange = (event: Event) => {
 </script>
 
 <template>
-  <label class="input">
+  <label class="input" :class="{ 'input--touched': isTouched }">
     <span class="input__label" :class="{ 'input__label--required': props.required }">
       {{ props.label }}
+      <!-- 
+        <span aria-label="required">*</span> 
+        should we use the above instead of the "after"?
+        see https://developer.mozilla.org/en-US/docs/Learn/Forms/Form_validation#full_example
+      -->
```

Comment:
To be discussed:
should we use the above instead of the "after"?
see [https://developer.mozilla.org/en-US/docs/Learn/Forms/Form_validation#full_example](https://developer.mozilla.org/en-US/docs/Learn/Forms/Form_validation#full_example)
```suggestion
        <span aria-label="required">*</span>
```

Reply:
not needed, required on the input itself is understandable enough.